guide/moderator/sticky.md

```diff
@@ -0,0 +1,16 @@
+## User Handbook
+
+### Sticky/Unsticky a thread
```

Comment:
This one is fixed, added the note about making sure the extension is enabled, and explained what the sticky is.